docs/topics/proc_show-debug-logging-local-spring-boot.adoc

```diff
@@ -0,0 +1,31 @@
+= Show Debug Logging on Localhost
+
+Deploy your application with the Fabric8 Maven Plugin `debug` option to enable debug logging.
+
+.Prerequisites
+* An application with debug logging.
+
+.Procedure
+.  Deploy your application with fabric8:debug to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean fabric8:debug
```

Comment:
This doesn't enable debug logging. This enables connecting a remote debugger. (Which is interesting, I didn't know about this possibility. CC @tradej)